### PR TITLE
style: fix button rtl.less

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/22239

补充 https://github.com/ant-design/ant-design/pull/22252

### 💡 Background and solution

- https://github.com/ant-design/ant-design/pull/22252/files#diff-cab8b58a530065e4a5dd1375a637ebcdR405-R410

I'm sorry for losing these.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | none          |
| 🇨🇳 Chinese |  none         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
@afc163 我点的merge 看的 commit 里面 好像不太一样，是不是有问题呢？

File: components/button/style/mixin.rtl.less [hunk 1]
```diff
@@ -1,4 +1,10 @@
 .btn-group(@btnClassName: btn) {
+  .@{btnClassName} + .@{btnClassName},
+  .@{btnClassName} + &,
+  span + .@{btnClassName},
+  .@{btnClassName} + span,
+  > span + span,
+  & + .@{btnClassName},
   & + & {
```
Comment (yoyo837) [inline components/button/style/mixin.rtl.less]: 看上去怪怪的🤨

Comment (xrkffgg) [reply components/button/style/mixin.rtl.less]: 这是原来的，我改的时候，这段给漏了，这个 PR 给补上

**Comment by yoyo837** [comment]
选 Squash

**Comment by xrkffgg** [comment]
尴尬了，第一次点，有办法取消吗？

**Comment by yoyo837** [comment]
Squash 会把一个PR的多个commit 合并成一个，merge不会。

本次就算了呗。而且只有一个commit.

**Comment by xrkffgg** [comment]
有 2 个 😅😅  
影响了 commit o(╥﹏╥)o
@afc163 
I'm very sorry.

**Comment by xrkffgg** [comment]
@yoyo837 还有哪些地方要特别注意的吗？

**Pull request merged**